barretenberg/cpp/src/barretenberg/stdlib/plonk_recursion/aggregation_state/aggregation_state.hpp

```diff
@@ -16,39 +17,44 @@ template <typename Curve> struct aggregation_state {
     typename Curve::Group P1;
 
     // The public inputs of the inner circuit are now private inputs of the outer circuit!
-    std::vector<typename Curve::ScalarField> public_inputs;
-    AggregationObjectIndices proof_witness_indices;
+    // std::vector<typename Curve::ScalarField> public_inputs;
+    // AggregationObjectIndices proof_witness_indices;
```

Comment:
proof_witness_indices are just the 16 indices of P0 and P1, so definitely not needed.